### PR TITLE
[#10197] More retriers when requesting balances

### PR DESCRIPTION
**Pull request by rasom**
fix (well rather hide better) #10197 

status: ready

### CODE REVIEW

**Comment by rasom** [comment]
@Serhy 
> With the latest nightly it reproduced probably 1 of 7 relogins.

Can you please check if it is better this way? :)

File: src/status_im/wallet/core.cljs [hunk 1]
```diff
@@ -34,7 +34,7 @@
    {:method            "eth_getBalance"
     :params            [address "latest"]
     :on-success        on-success
-    :number-of-retries 10
+    :number-of-retries 50
```
Comment (flexsurfer) [inline src/status_im/wallet/core.cljs:37]: O_o

Comment (rasom) [reply src/status_im/wallet/core.cljs:37]: yeah, I'm really tired of it

File: src/status_im/ethereum/json_rpc.cljs [hunk 2]
```diff
@@ -171,11 +172,20 @@
    "mailservers_getChatRequestRanges" {}
    "mailservers_deleteChatRequestRange" {}})
 
-(defn on-error-retry [call-method {:keys [method number-of-retries on-error] :as arg}]
+(defn on-error-retry
+  [call-method {:keys [method number-of-retries delay on-error] :as arg}]
```
Comment (flexsurfer) [inline src/status_im/ethereum/json_rpc.cljs]: do we use delay?

Comment (flexsurfer) [inline src/status_im/ethereum/json_rpc.cljs:176]: `delay` is not used currently ?

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ethereum/json_rpc.cljs (hunk 2 above)
Comment (rasom) [reply src/status_im/ethereum/json_rpc.cljs:176]: see line 179

**Comment by Serhy** [comment]
We with @churik tested and could not reproduce originnal issue now with this PR builds on iOS and Android. 
I could not reproduce bug in Release1.2 build as well actually today (~20 of 20 successful attempts made)... 
Failed autotests are knows GHI (apart   test_refresh_button_browsing_app_webview which is not knnown but exists in develop too)

I don't see any obstacles to not merge it! Thanks @rasom !

**Pull request merged**